### ISSUE TITLE
Missing files used in the docs

### ISSUE THREAD

**Issue by hcastel** [pyrfume/pyrfume#50]
Hi,

I didn't know whether to put this here or in pyrfume-data since it concerns both the docs and the data.

The doc page [3. Featurization for Machine Learning](https://github.com/pyrfume/pyrfume/blob/main/docs/featurization.ipynb) presents the use of pre-computed molecular features with
```python
morgan_sim = pyrfume.load_data("morgan/features_sim.csv", cids=cids)
mordred = pyrfume.load_data("mordred/features.csv", cids=cids)
```
However, neither of those two files exist in the [Morgan](https://github.com/pyrfume/pyrfume-data/tree/main/morgan) and [Mordred](https://github.com/pyrfume/pyrfume-data/tree/main/mordred) repositories, only `cids.csv` files and `main.ipynb` notebooks that are supposed to produce those files but seem to have failed.

Is this a deprecated feature, a bug, or is the documentation not up to date and there is a new way for doing this?

Also, the link at the bottom to [NSPDK](https://dtai.cs.kuleuven.be/drupal/software/nspdk) is broken (and there are typos on "filtred" and "favoriate").

**Comment by rgerkin** [comment]
@TravisGould Can you handle this?

**Comment by TravisGould** [comment]
Yup, I'll look into it.

**Comment by rgerkin** [comment]
The docs issue has been fixed in #51.  The .csv files for Mordred and Morgan are very large and need to be recomputed and then externally linked, the machinery for which is handled in pyrfume/pyrfume-data#184, and the files for which will be re-shared shortly.

**Comment by rgerkin** [comment]
Fixed in https://github.com/pyrfume/pyrfume-data/pull/184, and molecules lists updated in https://github.com/pyrfume/pyrfume-data/pull/185.  A PR to compute updated mordred or morgan fingerprints is welcome.
